EIPS/eip-3643.md

```diff
@@ -337,17 +322,16 @@ interface ITrustedIssuersRegistry {
     function getTrustedIssuers() external view returns (IClaimIssuer[] memory);
     function isTrustedIssuer(address _issuer) external view returns(bool);
     function getTrustedIssuerClaimTopics(IClaimIssuer _trustedIssuer) external view returns(uint[] memory);
+    function getTrustedIssuersForClaimTopic(uint256 claimTopic) external view returns (IClaimIssuer[] memory);
     function hasClaimTopic(address _issuer, uint _claimTopic) external view returns(bool);
```

Comment:
To enhance clarity and consistency, I suggest renaming "TrustedIssuersRegistry" to "ClaimIssuersRegistry", it makes sense since the registry is supposed to contain "ClaimIssuer"

This change will make the connections with "ClaimTopicsRegistry" more logical and intuitive. 

Currently, we use both "ClaimIssuersRegistry" and "TrustedIssuersRegistry" to refer to the same concept, which can confuse. 

Additionally, it aligns well with the purpose of the "ClaimIssuersRegistry" to contain information about claim issuers. This adjustment improves comprehension and coherence within the code and documentation.

Reply:
The denomination Trusted Issuers makes more sense to people coming from a finance background, and was already implemented in T-REX for a long time, renaming it now would be problematic for compatibility with all Tokens already issued and doesn't serve any important purpose.